plugins/inventory/kubevirt.py

```diff
@@ -16,23 +16,23 @@
 - "KubeVirt.io Project (!UNKNOWN)"
 
 description:
-- Fetch running VirtualMachineInstances for one or more namespaces with an optional label selector.
-- Groups by namespace, namespace_vmis and labels.
-- Uses the kubectl connection plugin to access the Kubernetes cluster.
-- Uses *.kubevirt.(yml|yaml) YAML configuration file to set parameter values.
+- Fetch running C(VirtualMachineInstances) for one or more namespaces with an optional label selector.
+- Groups by cluster name, namespace and labels.
+- Uses the M(kubernetes.core.kubectl) connection plugin to access the Kubernetes cluster.
+- Uses V(*.kubevirt.[yml|yaml]) YAML configuration file to set parameter values.
```

Comment:
You can also use round brackets with `V`:
```suggestion
- Uses V(*.kubevirt.\\(yml|yaml\\)) YAML configuration file to set parameter values.
```
(To use single backslashes, convert `DOCUMENTATION = """` to `DOCUMENTATION = r"""`.)

Reply:
I changed it to brackets to be consistent with the start of `EXAMPLES`. Actually, I liked brackets better. Thanks for the suggestion though.